irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -122,9 +122,13 @@ namespace iroha {
 
      protected:
       const std::string init_ = R"(
+CREATE TABLE IF NOT EXISTS role (
+    role_id character varying(45),
+    PRIMARY KEY (role_id)
+);
 CREATE TABLE IF NOT EXISTS domain (
     domain_id character varying(164),
-    default_role character varying(164),
+    default_role character varying(45) NOT NULL REFERENCES role(role_id),
```

Comment:
that is good! I had this question in validation process, so I guess you can delete this task from the backlog